frontend-project/src/pages/letters/list/index.tsx

```diff
@@ -2,15 +2,21 @@ import { ProColumns } from '@ant-design/pro-table';
 import React, { FC } from 'react';
 import { formatMessage } from 'umi-plugin-react/locale';
 
-import { ChannelName } from '@/components/Table/ChannelName';
 import { Letter, fetchLettersPage } from '@/services/letters';
 import Table from '@/components/Table';
-
+import { ChannelName } from '@/components/Table/ChannelName';
 import { CaseName } from '@/components/Table/CaseName';
 import { InstitutionName } from '@/components/Table/InstitutionName';
+import { DocumentTypeName } from '@/components/Table/DocumentTypeName';
 
 const TableList: FC<{}> = () => {
   const columns: ProColumns<Letter>[] = [
+    {
+      title: formatMessage({ id: 'letters-list.table.columns.documentType.title' }),
+      dataIndex: 'documentType',
+      render: (documentType: number) =>
+        typeof documentType === 'number' ? <DocumentTypeName id={documentType} /> : documentType,
```

Comment:
Jak rozumiem @ad-m typ dokumentu też (zanim nie wyjdzie z kancelarii/poczekalni) jest nie ustawiony? i stąd to zabezpieczenie?

Reply:
Tak